src/components/Cohort/Documents/PdfRenderer/PdfRenderer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Document, Page, Text, View, StyleSheet } from '@react-pdf/renderer'
+
+// Create styles
+const styles = StyleSheet.create({
+  page: { flexDirection: 'column' },
+  section: {
+    display: 'flex',
+    flexDirection: 'column',
+    margin: 4,
+    padding: 5,
+    flexGrow: 1
+  },
+  header: {
+    marginBottom: 40,
+    padding: 30,
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    borderBottom: '2px solid #153D8A',
+    borderTop: '2px solid #153D8A',
+    fontSize: '1.3rem'
+  }
+})
+
+type PdfRendererProps = {
+  content?: string
+}
+
+const PdfRenderer: React.FC<PdfRendererProps> = ({ content }) => {
+  if (!content) return <p></p>
```

Comment:
maybe make content required and check if there is one before calling PdfRenderer

Reply:
this is what I started to do in the other component but then I forgot to do it here 😓